src/Analyzers/Core/Analyzers/UseNullPropagation/AbstractUseNullPropagationDiagnosticAnalyzer.cs

```diff
@@ -13,9 +14,17 @@
 
 namespace Microsoft.CodeAnalysis.UseNullPropagation;
 
-internal static class UseNullPropagationConstants
+internal static class UseNullPropagationHelpers
 {
+    public const string IsTrivialNullableValueAccess = nameof(IsTrivialNullableValueAccess);
     public const string WhenPartIsNullable = nameof(WhenPartIsNullable);
+
+    public static bool IsSystemNullableValueProperty([NotNullWhen(true)] ISymbol? symbol)
+        => symbol is
+        {
+            Name: nameof(Nullable<int>.Value),
```

Comment:
>Name: nameof(Nullable<int>.Value), [](http://example.com/codeflow?start=12&length=34)

Does the SpecialType check prevent the need for the namespace checks?

Reply:
Yes